.gitignore

```diff
@@ -18,4 +18,7 @@ qmcpy/workouts/Outputs/*.png
 qmcpy/__pycache__/
 qmcpy/sphinx/_build/
 qmcpy/sphinx/_static/
-qmcpy/sphinx/_templates/
\ No newline at end of file
+qmcpy/sphinx/_templates/
+
+# Virtual environment stuff
+venv
```

Comment:
Not sure if people are using virtual environments, but I generally use it when I have a new project.  We could consider suggesting it to customers in the `README`.

Reply:
I use pip with virualenv and virtualenvwrapper. I think everyone is set up with a virtual environment for the project, but we can confirm at the next meeting. 

I added the following to the end of qmcpy/README.md:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
# Tips
It may be helpful to create a virtual environment for this project<br>
Packages and versions are compiled in requirements.txt<br>
In order for the software to find packages, make sure .../qmcpy/ is in your path